src/EA.Weee.Web/Areas/AatfReturn/Views/SelectYourPCS/Reselect.cshtml

```diff
@@ -0,0 +1,59 @@
+﻿@using System.Web.Mvc.Html
+@using EA.Prsd.Core.Web.Mvc.RazorHelpers
+@using EA.Weee.Core.AatfReturn
+@using EA.Weee.Web.Infrastructure
+@model EA.Weee.Web.Areas.AatfReturn.ViewModels.SelectYourPcsViewModel
+@{
+    ViewBag.Title = "Select Your PCS";
+}
+
+    <section id="main-content">
```

Comment:
Do we need a duplicate of the index view. Not quite sure what this is trying to do.

Reply:
Removed this view and updated the Index view